### PR TITLE
samples: wifi: Initialize wifi_connect_req_params structure

### PR DESCRIPTION
**Pull request by D-Triveni**
This fix addresses the issue where update to the channel value was not taking effect.

Fixes SHEL-2417.

### CODE REVIEW

**Review by krish2718 — COMMENTED** [review]
The commit log should explain the root cause not just the issue description.

**Comment by D-Triveni** [comment]
> The commit log should explain the root cause not just the issue description.

Updated.

**Review by chiranjeevi2776 — APPROVED** [review]

**Review by bama-nordic — APPROVED** [review]

**Review by muraliThokala — APPROVED** [review]

File: samples/wifi/raw_tx_packet/src/wifi_connection.c [hunk 1]
```diff
@@ -196,7 +196,7 @@ static int __wifi_args_to_params(struct wifi_connect_req_params *params)
 static int wifi_connect(void)
 {
 	struct net_if *iface = NULL;
-	static struct wifi_connect_req_params cnx_params;
+	static struct wifi_connect_req_params cnx_params = { 0 };
```
Comment (krish2718) [inline samples/wifi/raw_tx_packet/src/wifi_connection.c:199]: Why is `static` in use in the first place? I don't see any necessity.

**Review by krish2718 — CHANGES_REQUESTED** [review]
Also, I don't understand how this can cause SHEL-2417, there the band and channel are being overwritten and not relying on initial values.

File: samples/wifi/raw_tx_packet/src/wifi_connection.c (hunk 1 above)
Comment (krish2718) [reply samples/wifi/raw_tx_packet/src/wifi_connection.c:199]: And just initialising zero doesn't mean that the values are sane, you need to have a `set_defaults` which you did in https://github.com/nrfconnect/sdk-nrf/pull/13715

Comment (D-Triveni) [reply samples/wifi/raw_tx_packet/src/wifi_connection.c:199]: > Why is `static` in use in the first place? I don't see any necessity.

Will remove and raise new PR.

**Comment by D-Triveni** [comment]
> Also, I don't understand how this can cause SHEL-2417, there the band and channel are being overwritten and not relying on initial values.

Closing this PR. Will do necessary changes as per the discussion on SHEL-2417

**Pull request closed without merging**